---
title: "Curriculum Vitae"
---

## Introduction

::: {.cv}

PhD in plant molecular genetics, 8 years experience in data intensive fields both in academic and private sector.

Experienced R programmer. Strong knowledge of statistics and data analysis. Expert in data visualization, data reporting, data communication, data driven products.

:::

## Work Experience

:::: {.grid .cv style="--bs-columns: 6;"}

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Data Scientist

*[Accurat](https://accurat.it/) <br> Milan, Italy* <br>
**Sep 2019 - Nov 2022**

- Scientific support to the data visualization design teams.
- Data intelligence, data driven reporting.
- Development of internal tools in R, development of data workflow.
- Privacy and GDPR compliance.
- Development of data science classes for Feltrinelli Education.
- Development of commercial offers for technical clients.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Postdoc

*[Institut de Recherche pour le Développement (IRD)](https://www.ird.fr/) <br> Montpellier, France* <br>
**Jan 2018 - Dec 2018**

Functional genomics of wild and cultivated rice species.

- Development of an analytical and graphical pipeline for comparing transcriptomes of different rice strains and species.
- Development of an R package for graph-based comparison of rice panicle topology.


:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Postdoc

*[CEPLAS/Heinrich Heine University](https://www.ceplas.eu/en/home/) <br> Dusseldorf, Germany* <br>
**Apr 2015 - Nov 2017**

Evolution and synthetic biology of C4 photosynthesis in plants.

- Development of analytic and modeling pipelines for comparing transcriptomes of different plant species.
- Analysis of proteomic data.
- Development of gene editing experiments for engineering of photosynthetic pathways.
- Setup of the Food Security area of competence in collaboration with the University of Cologne.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Visiting Scientist

*[North Carolina State University](https://www.ncsu.edu/) <br> Raleigh, NC, USA* <br>
**Aug 2014 - Oct 2014**

Fruit development.

- Setup of Chromatin Immunoprecipitation protocols in [Professor Robert Frank’s Lab](https://frankslab.wordpress.ncsu.edu/).

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  PhD Candidate

*[University of Milan](https://www.unimi.it/en) <br> Milan, Italy* <br>
**Nov 2011 - Nov 2014**

Flower development.

- Analysis and modeling of transcriptomic data,
- Development of protocols for Laser microdissection of plant tissues.
- Development of gene editing protocols.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Museum Guide

*[Museo Nazionale della Scienza e Tecnologia](https://www.museoscienza.org/en) <br> Milan, Italy* <br>
**Feb 2010 - Oct 2011**

While studying for my master degree, in the weekend I gave guided tours in sections about:

- Air, land and water transports.
- Submarine Enrico Toti

:::

::::

## Teaching Experience

::: {.cv}

<br>

See section: [Trainings](trainings.html).

<br>

:::

## Digital Skills

:::: {.grid .cv style="--bs-columns: 6;"}

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  R / Python

Expert in programming for statistics and data science.

- Tidyverse / Tidymodels for dataviz data wrangling and data modeling.
- Literate programming in Rmarkdown, Jupyter, and Quarto.
- R package development.
- Continuous integration of end to end analytic workflows.
- Basis of Pandas and Scikit-Learn.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Web development

Web development dedicated to communicating data and scientific results.

- Reporting, publishing and dashboarding with Quarto.
- Web native dataviz with D3 and Observable.
- Basis of generic web development.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  Else

- Open Access.
- Git / Github.
- Continuos Integration.
- Docker.

:::

::::

## Main Education

:::: {.grid .cv style="--bs-columns: 6;"}

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  PhD in Biomolecular Sciences

*Università degli Studi di Milano.*<br>
*Department of Biosciences.* <br>
*Milan, Italy.*

Awarded in **November 2014**.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  MSc in Molecular Biotechnology

*Università degli Studi di Milano.*<br>
*Department of Biosciences.* <br>
*Milan, Italy.*

Awarded in **October 2011**.

:::

::: {.g-col-6 .g-col-sm-3 .g-col-xxl-2}

###  BSc in Industrial and Environmental Biotechnology

*Università degli Studi di Milano.*<br>
*Department of Biosciences.* <br>
*Milan, Italy.*

Awarded in **October 2009**.

:::

::::

## Additional Education

:::: {.grid .cv style="--bs-columns: 6;"}

::: {.g-col-6 .g-col-md-3}

- Design Thinking - *Protostart, Düsseldorf, September 2017*
- Data Mining - gene expression data evaluation - *Heinrich Heine University, November 2016*
- How to successfully communicate your research - *Cologne University, December 2015* 
- How to lead and coach a PhD candidate - *forschungszentrum jülich - December 2015*

:::
::: {.g-col-6 .g-col-md-3}


-  Advanced Statistics for the Life Sciences - *EDX, September 2015*
- Introduction to Linear Models and Matrix Algebra - *EDX, May 2015*
- Statistics and R for the life Sciences - *EDX, May 2015*
- Getting and Cleaning Data - *Coursera, August 2014*
- R programming *Coursera, July 2014 *

:::

::::

## Languages

::: {.cv}

Italian (native) -
English C2 (fluent) -
German A1 (beginner) -
French A1 (beginner)

:::

## Peer Reviewed Publications

:::: {.grid .bib style="--bs-columns: 6;"}

::: {.g-col-6 .g-col-md-3}

Hahn, F., Eisenhut, M., Mantegazza, O., & Weber, A. P. (2018). Homology-directed repair of a defective glabrous gene in arabidopsis with Cas9-based gene targeting. Frontiers in Plant Science, 9, 424.

Mantegazza, O., Gregis, V., Chiara, M., Selva, C., Leo, G., Horner, D. S., & Kater, M. M.(2014). Gene coexpression patterns during early development of the native arabidopsis reproductive meristem: Novel candidate developmental regulators and patterns of functional redundancy. The Plant Journal, 79 (5), 861–877.

Lundquist, P. K., Mantegazza, O., Stefanski, A., Stühler, K., & Weber, A. P. (2017). Surveying the oligomeric state of arabidopsis thaliana chloroplasts. Molecular Plant, 10 (1), 197–211.

Hahn, F., Mantegazza, O., Greiner, A., Hegemann, P., Eisenhut, M., & Weber, A. P. (2017). An efficient visual screen for CRISPR/Cas9 activity in arabidopsis thaliana. Frontiers in Plant Science, 8, 39. 2/35.

:::

::: {.g-col-6 .g-col-md-3}

Schuler, M. L., Mantegazza, O., & Weber, A. P. (2016). Engineering C4 photosynthesis into C3 chassis in the synthetic biology age. The Plant Journal, 87 (1), 51–65.

Mantegazza, O., Gregis, V., Mendes, M. A., Morandini, P., Alves-Ferreira, M., Patreze, C. M., Nardeli, S. M., Kater, M. M., & Colombo, L. (2014). Analysis of the arabidopsis REM gene family predicts functions during flower development. Annals of Botany, 114 (7), 1507–1515.

Hahn, F., Eisenhut, M., Mantegazza, O., & Weber, A. P. (2017). Generation of targeted knockout mutants in arabidopsis thaliana using CRISPR/Cas9. BIO-PROTOCOL, 7 (13).

Harrop, T. W., Mantegazza, O., Luong, A. M., Béthune, K., Lorieux, M., Jouannic, S., & Adam, H. (2019). A set of AP2-like genes is associated with inflorescence branching and architecture in domesticated rice. Journal of Experimental Botany, 70 (20), 5617–5629.


:::

::::
